agent_plugin_builder/agent_plugin_builder.py

```diff
@@ -70,3 +75,9 @@ def _log_arguments(args):
 def _setup_logging(verbosity):
     reset_logger()
     setup_logging(verbosity)
+
+
+def _get_source_dir(source_dir: str | None, agent_plugin_manifest: AgentPluginManifest) -> str:
```

Comment:
It doesn't look like the `source_dir` can be `None`.

Reply:
It's `None` [by default](https://github.com/guardicode/agent-plugin-builder/blob/adf0b73728d9ea213e0d5c9c55b43bf55d8a56b6/agent_plugin_builder/agent_plugin_builder.py#L36), if the user doesn't specify `--source_dir`

Reply:
Oh, because it's from `argparse`, not the options object. Got it.